keyboards/lily58/keymaps/lily58l_vii33/oled.c

```diff
@@ -0,0 +1,374 @@
+                                                         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0x40, 0x40, 0x40, 0x40, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xc0, 0x40, 0x40, 0x80, 0x00, 0x80, 0x40, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x1e, 0x21, 0xf0, 0x04, 0x02, 0x02, 0x02, 0x02, 0x03, 0x02, 0x02, 0x04, 0x04, 0x04, 0x03, 0x01, 0x00, 0x00, 0x09, 0x01, 0x80, 0x80, 0xab, 0x04, 0xf8, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x03, 0x1c, 0x20, 0x20, 0x3c, 0x0f, 0x11, 0x1f, 0x02, 0x06, 0x18, 0x20, 0x20, 0x38, 0x08, 0x10, 0x18, 0x04, 0x04, 0x02, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00,
+                                                     },
+
+                                                     /* 'sneak2', 32x22px */
+                                                     {
+                                                         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0x40, 0x40, 0x40, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xe0, 0xa0, 0x20, 0x40, 0x80, 0xc0, 0x20, 0x40, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x3e, 0x41, 0xf0, 0x04, 0x02, 0x02, 0x02, 0x03, 0x02, 0x02, 0x02, 0x04, 0x04, 0x02, 0x01, 0x00, 0x00, 0x00, 0x04, 0x00, 0x40, 0x40, 0x55, 0x82, 0x7c, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x3f, 0x20, 0x30, 0x0c, 0x02, 0x05, 0x09, 0x12, 0x1e, 0x04, 0x18, 0x10, 0x08, 0x10, 0x20, 0x28, 0x34, 0x06, 0x02, 0x01, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
+                                                     }};
+
+    // current_wpm   = get_current_wpm(); 
+    
+    /* animation timer */
+    if (timer_elapsed32(anim_timer) > ANIM_FRAME_DURATION) {
+      anim_timer = timer_read32();
+              
+      /* jump */
+      if (lunaIsJumping || !lunaShowedJump) {
+          /* clear */
+          oled_set_cursor(LUNA_X, LUNA_Y + 2);
+          oled_write("     ", false);
+
+          oled_set_cursor(LUNA_X, LUNA_Y - 1);
+
+          lunaShowedJump = true;
+
+      } else {
+          /* clear */
+          oled_set_cursor(LUNA_X, LUNA_Y - 1);
+          oled_write("     ", false);
+
+          oled_set_cursor(LUNA_X, LUNA_Y);
+      }
+
+      /* switch frame */
+      current_frame = (current_frame + 1) % 2;
+
+      /* current status */
+      if (lunaIsBarking) {
+          oled_write_raw_P(bark[abs(1 - current_frame)], ANIM_SIZE);
+
+      } else if (lunaIsSneaking) {
+          oled_write_raw_P(sneak[abs(1 - current_frame)], ANIM_SIZE);
+
+      } else if (current_wpm <= MIN_WALK_SPEED) {
+          oled_write_raw_P(sit[abs(1 - current_frame)], ANIM_SIZE);
+
+      } else if (current_wpm <= MIN_RUN_SPEED) {
+          oled_write_raw_P(walk[abs(1 - current_frame)], ANIM_SIZE);
+
+      } else {
+          oled_write_raw_P(run[abs(1 - current_frame)], ANIM_SIZE);
+      }
+    }
+}
+/* ------ LUNA PET END ------ */
+
+
+/* Remders Mario image. This costs a little bit of memory since two times a 512 byte image is rendered. */
+void render_mario(void) {
```

Comment:
As cool as this may be... it needs to be removed.   Nintendo is famously litigious,

Reply:
Sad, but ok. Replaced it with default Lily58L Logo from other keymaps.